packages/serverpod/lib/src/generated/protocol.dart

```diff
@@ -1806,13 +1812,6 @@ class Protocol extends _i1.SerializationManagerServer {
           .map((e) => deserialize<_i59.TableDefinition>(e))
           .toList() as dynamic;
     }
-    if (t == _i1.getType<List<_i59.DatabaseMigrationVersion>?>()) {
```

Comment:
Question: Hmm, why is this removed?

Reply:
probably because it is none-nullable now.